model/project.go

```diff
@@ -475,7 +475,8 @@ type TaskGroup struct {
 	Tasks                 []string        `yaml:"tasks" bson:"tasks"`
 	Tags                  []string        `yaml:"tags,omitempty" bson:"tags"`
 	// ShareProcs causes processes to persist between task group tasks.
-	ShareProcs bool `yaml:"share_processes" bson:"share_processes"`
+	ShareProcs      bool `yaml:"share_processes" bson:"share_processes"`
+	ContinueOnError bool `yaml:"continue_on_error" bson:"continue_on_error"`
```

Comment:
Nit: I think ContinueOnError should be in the "data about the task group" section, otherwise it implicitly looks related to ShareProcs.